### DISCUSSION TITLE
Welcome to DDF-RA Discussions!

### DISCUSSION THREAD

**Discussion by drewcdisc** [cdisc-org/DDF-RA#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  📢 **Announce to your community** that Discussions is available! Go ahead and send that tweet, post, or link it from the website to drive traffic here.

  🔗 If you use issue templates, **link any relevant issue templates** such as questions and community conversations to Discussions. Declutter your issues by driving community content to where they belong in Discussions. If you need help, here's a [link to the documentation](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

  ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by srakov-mtx** [comment]
Hi, I am trying to figure out how to represent a clinical trial that has multiple arms, each of which has a different cohort size, number of doses, and dose regimen.  I was going to put each dose in as an `InvestigationalIntervention`, which connects to an `Estimand`, but I don't see a connection from there to a `StudyArm`.  I would then put the dose regimen in as a `Timeline`, with the dosing itself as an `Activity`.  But I don't see a way to connect the `StudyArm` to either a specific timeline or to a specific `InvestigationalIntervention`.  Am I missing something?  Do I have to model these as different `StudyDesign`s against a single `Study`?  Do you recommend a different way of doing it?  Please advise.  Thank you!

**Reply by BSnoeijerCD** [reply]
Hi,
Thank you for your question. 
You are right that each dosing should be an activity. In that way it can be represented in the schedule of activity. 
In the current V2.0 USDM model you can link StudyArm to the corresponding timeline via the studyCell, studyEpoch and the corresponding encounters. Encounters are referred to in the timeline from ScheduledActivityInstance.  

For the next phase we are looking into relating the InvestigationalIntervention to both procedure and studyElement. In that way it can be both linked to the corresponding timings in the timeline (via procedure) and to StudyArm via studyElement.

**Reply by srakov-mtx** [reply]
Thank you for your reply!  I have another question.  How would you recommend I connect a `StudyArm` to a `StudyPopulation`, so I can capture the cohort id and size?  Thanks for your help!

**Comment by srakov-mtx** [comment]
Hi @BSnoeijerCD , many thanks for your response to my question above, it is very helpful!  I have another question, which I noted in my reply above; how would you recommend I capture the cohort id and size for a `StudyArm`?  I am thinking that I will connect  `StudyArm` to `StudyDesignPopulation`.  What do you think?  

Thanks for your help!

**Reply by BSnoeijerCD** [reply]
Hi,
Yes. For v2.0 you can do that. In the future models we are looking into linking this to the cohorts that we will be adding. In v2.0 this is not yet accounted for.

**Comment by srakov-mtx** [comment]
Hi, thanks for your comments above.  If you are willing to indulge me, I have more questions.

1. We have actually gone in a slightly different direction than I anticipated above.  Instead of creating a dose regimen as a separate `ScheduleTimeline`, we would like to integrate the dose regimen into the timeline as an `Activity` with `ScheduledActivityInstance` objects that represent the actual relationship between the `Dose`, the `Encounter`, and the `Timing`.  Do you think this makes sense as a model? 
2. If I wanted to put more metadata around the dose regimen activity, where would I put it?  In a new field in the `Activity` object?
3. How do you model a dosing `Activity` that could be either study drug or placebo?  The way I did it was to have the `Activity` have two conditional `Procedure`s underneath, one of which was "Study Injection" while the other was "Placebo Injection."  Is this the right way to model it?  What I wanted was to be able to connect each procedure to the correct `InvestigationalIntervention`.  

Please let me know what you think.  Thank you!

best
Simon Rakov

**Comment by BSnoeijerCD** [comment]
Hi Simon @srakov-mtx ,

Good to hear that you are working actively with the model. Please find my answers below:
1. I am not sure whether I understand correctly. You can add a timeline to an activity as well. So an activity Dosing can have a subtimeline with the corresponding regimen details using the timing class to define the expected time between each dose. For regimens that works well.
2. Do you have an example of what you want to map? In version 2.0 I think the subtimeline linked to activity should be sufficient. Furthermore you can link to a corresponding dosing procedure. Version 2.6 has new attributes that enables you to link this procedure to the corresponding intervention details and the corresponding administration details like dose, route and frequency.
3. Yes. I would do it in the same way.

**Reply by srakov-mtx** [reply]
Thank you!  That's very helpful.  Here are my responses, and some follow-up questions.

1. I missed that you could connect an activity to a sub-timeline.  Thanks for pointing that out, that seems perfect.  We have studies where the dosing is represented in the study's main schedule of activities, i.e the schedule shows encounters/visits at 0 and 8 weeks, each of which has a Dosing activity where you dose the drug or placebo.  That's the dose regimen for the drug.  
   1. If the dose regimen is represented in the main timeline in this way, would you recommend creating a sub-timeline for the Dosing activity, and duplicating that dose regimen information into it?  
1. I wanted to put a name on the dosing regimen.  I can do that now, I think, by naming the sub-timeline.
   1. Where can I find model 2.6?
1. Great to hear!  Thanks.
1. One more question:  can you tell us what the reasoning is behind attaching the `InvestigationalIntervention` to the `StudyElement` instead of the `StudyCell`?

Thanks again!

**Comment by BSnoeijerCD** [comment]
Hi Simon,
1. I don't think it is necessary to duplicate the dosing in that case. If it is clear that it is only once at each visit with corresponding timing then that is enough.
2. If you are a CDISC volunteer, you can find the latest version in the GitHub main branch. We are now at 2.7: https://github.com/cdisc-org/DDF-RA/blob/main/Deliverables/UML/USDM_UML.png
3. The studyElement is what can happen during a period (studyEpoch) and in an arm (studyArm). The studyElement can be reused for example in a crossover study. The studyCell is a class that ties these three entities together.  So it only includes the relationships to studyElement, studyEpoch and studyArm. Think of it as a grid where the epochs are columns, the arms are rows and the elements are the potential contents of a cell.

**Reply by srakov-mtx** [reply]
Hi Berber,

Thanks for your reply!  I really appreciate it.  In response to (2), how do I become a CDISC volunteer, and what files would I get access to if I did that?  I have now noticed that I can see the 2.7.1 and 2.8.0 release branches.  Which brings me to the question, in the changelog files here:  https://github.com/cdisc-org/DDF-RA/tree/main/Deliverables/UML, why do the filenames go above version 2.8?

**Reply by BSnoeijerCD** [reply]
Good point. 2.8 is the latest version. We first aligned with sprint numbers instead of version numbers. We will look into that. 
I forwarded your question regarding becoming a volunteer.

**Reply by jowencdisc** [reply]
Hi Simon, 
In order to get you on-boarded to the DDF Volunteer SME team (gives access to the DDF WIKI and the DDF distribution list) can you please complete the following steps.

1) Create a free cdiscID website account via the following link, [Create an Account | CDISC](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Furldefense.proofpoint.com%2Fv2%2Furl%3Fu%3Dhttps-3A__www.cdisc.org_user_registration%26d%3DDwMGaQ%26c%3DeIGjsITfXP_y-DLLX0uEHXJvU8nOHrUK8IrwNKOtkVU%26r%3DqcW2nC0iPcfDoXcU9JQbHYyNY0wF5-aVz2J62Dxlxm4%26m%3DeoNSK71W_7uNqgLfbJBknB2hHyLF0c4jd4CIogwsS28%26s%3DVjGporGBXy9QT-mjabq6hEbNPsbYEfpdpUpsHB-VhZk%26e%3D&data=04%7C01%7Cjowen%40cdisc.org%7Cb500d78c3a0141bd2b2108d96c96811a%7C078244a1de674c9e9088986d7f110a37%7C0%7C0%7C637660215953851368%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=gzAHOLV6V0NJYPmjKdx3ygr1NPX7gYkzUgmi6O11CK8%3D&reserved=0).
If you already have an account, this step can be skipped.

2) Log in to the CDISC website using your account, [Log in | CDISC](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Furldefense.proofpoint.com%2Fv2%2Furl%3Fu%3Dhttps-3A__www.cdisc.org_user_login%26d%3DDwMGaQ%26c%3DeIGjsITfXP_y-DLLX0uEHXJvU8nOHrUK8IrwNKOtkVU%26r%3DqcW2nC0iPcfDoXcU9JQbHYyNY0wF5-aVz2J62Dxlxm4%26m%3DeoNSK71W_7uNqgLfbJBknB2hHyLF0c4jd4CIogwsS28%26s%3D2XP5mf9KOuABYB3VnZOkbxoaTUb239-ujDWhYVow_Ug%26e%3D&data=04%7C01%7Cjowen%40cdisc.org%7Cb500d78c3a0141bd2b2108d96c96811a%7C078244a1de674c9e9088986d7f110a37%7C0%7C0%7C637660215953861388%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ADhb7qqC97Qk56KfArlTww9xAM%2BesfJ4nmP8JCJJYOI%3D&reserved=0).
There is also a Log in link to this page in the upper right corner of the CDISC website.

3) Navigate to [Become a Volunteer | CDISC](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Furldefense.proofpoint.com%2Fv2%2Furl%3Fu%3Dhttps-3A__www.cdisc.org_volunteer_form%26d%3DDwMGaQ%26c%3DeIGjsITfXP_y-DLLX0uEHXJvU8nOHrUK8IrwNKOtkVU%26r%3DqcW2nC0iPcfDoXcU9JQbHYyNY0wF5-aVz2J62Dxlxm4%26m%3DeoNSK71W_7uNqgLfbJBknB2hHyLF0c4jd4CIogwsS28%26s%3D9gwNBtotCh9HKGYWm49kgPu0MtMIALDM0FT-HeS1dOM%26e%3D&data=04%7C01%7Cjowen%40cdisc.org%7Cb500d78c3a0141bd2b2108d96c96811a%7C078244a1de674c9e9088986d7f110a37%7C0%7C0%7C637660215953861388%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=8QYl1O7%2BVsJDovaoTlZu93QzdTIp7e7gwx%2Bksvt7N0I%3D&reserved=0), complete all steps, and submit your form.
Please select DDF from the list of projects and at the box at the bottom of the page specify the capacity of your involvement.
 
4) Wiki access will be provided shortly after completing these steps.
